invisible_cities/reco/icaro_components_test.py

```diff
@@ -0,0 +1,98 @@
+import numpy         as np
+import numpy.testing as npt
+import pandas        as pd
+
+from hypothesis.strategies import integers
+from hypothesis.strategies import floats
+from hypothesis            import given
+from pytest                import mark
+from pytest                import raises
+
+from   sklearn.linear_model import RANSACRegressor
+
+from .. core.testing_utils  import assert_dataframes_equal
+from .. core                import core_functions   as core
+
+from .                      import icaro_components as icarcomp
+
+
+@mark.parametrize("signal", (icarcomp.type_of_signal.nS1, icarcomp.type_of_signal.nS2))
```

Comment:
```suggestion
@mark.parametrize("signal", icarcomp.type_of_signal)
```
this is enough, I think